cmd/traffic_layout/engine.cc

```diff
@@ -200,7 +200,10 @@ clean_parent(const std::string &bin_path)
 {
   char cwd[MAX_CWD_LEN];
   getcwd(cwd, sizeof(cwd));
-  std::string RealBinPath = realpath(bin_path.c_str(), nullptr); // bin path
+
+  char resolved_binpath[MAX_CWD_LEN];
+  realpath(bin_path.c_str(), resolved_binpath); // bin path
```

Comment:
Shouldn't the return value of `realpath` be checked for `nullptr`?

Reply:
If we have the second argument ``resolved_binpath`` defined as char [], there is no need to check return value. There will be no nullptr. I can add a line of checking ``strlen()`` of ``resolved_binpath`` value zero to print out errno.